builtin/algorithm/mod.rs

```diff
@@ -1,31 +1,39 @@
 // Copyright © 2024 The µcad authors <info@ucad.xyz>
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
-use microcad_lang::{objects::*, resolve::*};
+use microcad_lang::{eval::Builtin, objects::*, resolve::*, syntax::Identifier, value::Value};
```

Comment:
Keep it short (we will make proper sub prelude spaces later).

```suggestion
use microcad_lang::{eval::*, objects::*, resolve::*, syntax::* value::*);
```